api/spec/requests/spree/api/v2/storefront/taxons_spec.rb

```diff
@@ -267,6 +267,7 @@
       let(:translated_slug) { 'test_slug_es' }
 
       before do
+        Spree::Locales::SetFallbackLocaleForStore.new.call(store: store2)
```

Comment:
Now that I'm looking at the tests - it would probably be wise to cover this fallback functionality with tests in general. I guess what we are missing is:
1. Test whether the locale controller helper sets the fallback when `current_store` is present.
2. Test whether the fallback plugin itself works as intended.

Reply:
1. This is covered e.g. by the requests tests for the API. I forgot to remove setting it manually here, it works without it. I moved away from generating the slugs with fallbacks manually, and it now fully relies on mobility, so the old assertions check this behavior end-to-end now.
2. I added two specs for that